src/check_ast.rs

```diff
@@ -348,6 +349,11 @@ where
                 if self.settings.enabled.contains(&CheckCode::B019) {
                     flake8_bugbear::plugins::cached_instance_method(self, decorator_list);
                 }
+                if self.settings.enabled.contains(&CheckCode::C901)
+                    || self.settings.mccabe.max_complexity > -1
```

Comment:
I'd vote to nix this "or", and just drive based on `self.settings.enabled`. Wdyt?